modules/minimum-required-permissions-ipi-azure.adoc

```diff
@@ -23,8 +23,6 @@ The following permissions are required for creating an {product-title} cluster o
 ====
 * `Microsoft.Authorization/policies/audit/action`
 * `Microsoft.Authorization/policies/auditIfNotExists/action`
-* `Microsoft.Authorization/roleAssignments/read`
-* `Microsoft.Authorization/roleAssignments/write`
```

Comment:
So now these two perms are technically "optional" but are required with the default install config settings...